docs/user-guide/dual-mesh.ipynb

```diff
@@ -0,0 +1,6253 @@
+{
```

Comment:
<p>This should be renamed:</p><p> </p><p>Computing the Dual Grid </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>